skyportal/handlers/api/comment.py

```diff
@@ -48,9 +57,29 @@ def get(self, comment_id):
               application/json:
                 schema: Error
         """
-        comment = Comment.get_if_accessible_by(
-            comment_id, self.current_user, raise_if_none=True
-        )
+
+        if comment_on_what is None:
+            comment_on_what = 'object'
+
+        if type(comment_on_what) != str:
+            return self.error(
+                f'Wrong input type {type(comment_on_what)} given as "comment_on_what" argument'
+            )
```

Comment:
```suggestion
```
I don't think this actually does anything. I believe Tornado will always just pass the argument parsed from the URL as a string. For example, if you did `GET /api/comments/{comment_id}/123`, then `comment_on_what = "123"` and not the int.  I think just have the error case with the unknown type in the `else` block below is sufficient.